lib/matplotlib/backends/backend_pdf.py

```diff
@@ -1040,8 +1040,9 @@ def embedTTFType42(font, characters, descriptor):
             for c in characters:
                 ccode = c
```

Comment:
The only suggestion I have is to clean this up; `ccode` can just be the name at the beginning of the loop.